packages/loot-core/src/client/actions/queries.ts

```diff
@@ -368,6 +373,101 @@ export function forceCloseAccount(accountId) {
   return closeAccount(accountId, null, null, true);
 }
 
+// Remember the last transaction manually added to the system
+export function setLastTransaction(
```

Comment:
Just curious, what was the rationale to split these actions out into a separate file specifically?

Reply:
I'm not sure, maybe @MatissJanis can answer. But Redux recommendations points to using feature folders which colocates reducers and actions https://redux.js.org/style-guide/#structure-files-as-feature-folders-with-single-file-logic

Reply:
Pre-open-source code, so sadly I won't know the reason here. But I would venture a guess that the rationale was - whatever was quicker & easier at the time without much thought for scaling/clean code.